tests/src/CLRTest.MockHosting.targets

```diff
@@ -0,0 +1,21 @@
+<!--
+***********************************************************************************************
+CLRTest.MockHosting.targets
+
+This file contains the logic for correctly hooking up a mock hostpolicy to a test.
+
+***********************************************************************************************
+-->
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <HostPolicyFileExtension>so</HostPolicyFileExtension>
+    <HostPolicyFileExtension Condition="$(TargetsOSX) =='true'">dylib</HostPolicyFileExtension>
```

Comment:
This condition is wrong. It should be `'$(TargetsOSX)' =='true'` (i.e. missing single quotes).

Reply:
Also please add the `.` in the extension, otherwise it is just a string.